ASM/c/item_table.h

```diff
@@ -22,6 +22,7 @@ typedef enum {
     /* 13 */ SILVER_CHEST,
     /* 14 */ SKULL_CHEST_SMALL,
     /* 15 */ SKULL_CHEST_BIG,
+    /* 16 */ HEART_CHEST,
```

Comment:
There should be small and big versions of the heart chest type that work like `SKULL_CHEST_SMALL` and `SKULL_CHEST_BIG`.

Reply:
I didn't make small and big versions because most chests don't have that variation, but I guess I could do that

Reply:
This change is a bit weird because `HEART_CHEST_BIG` goes unused. :man_shrugging:

Reply:
It shouldn't go unused. I'll send a quick PR to show what I mean.

Reply:
AngheloAlf#1